http_handler.go

```diff
@@ -22,8 +22,11 @@ func TracingHandlerFunc(h http.HandlerFunc) http.HandlerFunc {
 // Trace is middleware that wraps a http.Handler adding request-tracing and
 // profiling capabilities.
 // This function can be used to enhance a http router adding extra
-// capabilities to all its enpoints.
+// capabilities to all its endpoints.
 func Trace(handler http.Handler) http.Handler {
+	if handler == nil {
```

Comment:
How would it have been `nil` in the first place? Looks very much like defensive programming.

Isn't it initialized correctly when it starts?